adapters/criteo/criteo_test.go

```diff
@@ -1,27 +1,15 @@
 package criteo
 
 import (
-	"testing"
-
 	"github.com/prebid/prebid-server/adapters/adapterstest"
-	"github.com/prebid/prebid-server/config"
-	"github.com/prebid/prebid-server/openrtb_ext"
+	"testing"
 )
 
 func TestJsonSamples(t *testing.T) {
 
 	// Setup:
-	bidder, buildErr := builderWithGuidGenerator(
-		openrtb_ext.BidderCriteo,
-		config.Adapter{
-			Endpoint: "https://bidder.criteo.com/cdb?profileId=230",
-		},
-		config.Server{ExternalUrl: "http://hosturl.com", GvlID: 1, DataCenter: "2"},
-		newFakeGuidGenerator("00000000-0000-0000-00000000"),
-	)
-
-	if buildErr != nil {
-		t.Fatalf("Builder returned unexpected error %v", buildErr)
+	bidder := &adapter{
```

Comment:
Nitpick - Instead of creating a bidder directly, Builder method can be called to cover Builder method in the tests as follow :-

```
bidder, buildErr := Builder(openrtb_ext.BidderCriteo, config.Adapter{
		Endpoint: "https://bidder.criteo.com/openrtb/prebid/auction/request?profile=230"},
		config.Server{ExternalUrl: "http://hosturl.com", GvlID: 1, DataCenter: "2"})

	if buildErr != nil {
		t.Fatalf("Builder returned unexpected error %v", buildErr)
	}
```

Reply:
Thank you for the comment. I updated the pull request.